cmd/dag.go

```diff
@@ -172,3 +203,66 @@ func (o *ManifestOptions) Missing() error {
 
 	return err
 }
+
+// Info executes the dag info command
+func (o *DAGOptions) Info() (err error) {
+	info := &dag.Info{}
+	for _, refstr := range o.Refs {
+		s := &lib.DAGInfoParams{RefStr: refstr, Label: o.Label}
+		if err = o.DatasetRequests.DAGInfo(s, info); err != nil {
+			return err
+		}
+
+		var buffer []byte
+		switch strings.ToLower(o.InfoFormat) {
+		case "json":
+			if !o.Pretty {
+				buffer, err = json.Marshal(info)
+			} else {
+				buffer, err = json.MarshalIndent(info, "", " ")
+			}
+		case "yaml":
+			buffer, err = yaml.Marshal(info)
+		// case "cbor":
+		// 	buffer, err = info.MarshalCBOR()
+		default:
```

Comment:
Could you change this to `case "":`, and add a default case that returns an error that does `fmt.Errorf("unknown format %s", o.InfoFormat)`. This way we'll catch spelling mistakes like `jsonn`, treating them as errors instead of the default format.